gcp-instance-reaper/filter-instances.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+import datetime
+import re
+
+from nebula_sdk import Interface, Dynamic as D
+
+
+ni = Interface()
+
+# The `MINUTES_TO_WAIT` global variable is the number of minutes to wait for
+# a termination_date label to appear for the GCP instance. Please note that the
+# AWS Lambdas are limited to a 5 minute maximum for their total run time.
```

Comment:
> `AWS Lambdas`

Probably should just remove this sentence around lambdas.

Reply:
Good find. I thought I removed all references to AWS...